### DISCUSSION TITLE
ScenarioDef 관련 질문

### DISCUSSION THREAD

**Discussion by zilrty** [RimWorldKorea/RMK#688] [질문]
알티로 xlsx -> xml 변환하면 

xxx.label
xxx.description

얘네들 자동으로

xxx.scenario.name
xxx.scenario.description

이렇게 바뀜?

**Comment by RimWorldKorea** [comment]
추출할땐 그렇게 되는데 xlsx->xml도 그랬는진 모르겠네

그렇게 안되게 하기로 했던 것 같은데 확인해봐야할 듯

**Reply by zilrty** [reply]
보고서에 나오는거 거슬려서 

라벨이랑 디스크립션 추가하고 확인하니까 자꾸 나와서

xml 까보니까 시나리오 네임이랑 디스크립션이 2개씩 있음 ㅋㅋ
